README.md

```diff
@@ -1,135 +1,96 @@
+    eigenda-memstore-devnet-clean
+    eigenda-memstore-devnet-configs
+    eigenda-memstore-devnet-failback
+    eigenda-memstore-devnet-failover                 # to failover to ethDA. Use `eigenda-memstore-devnet-failback` to revert.
+    eigenda-memstore-devnet-grafana
+    eigenda-memstore-devnet-restart-batcher          # Restart batcher with new flags or image.
+    eigenda-memstore-devnet-start                    # EigenDA devnet that uses the eigenda-proxy in memstore mode (simulates an eigenda network but generates random certs)
+    eigenda-memstore-devnet-sync-status
+    eigenda-memstore-devnet-test
+```
 
-The full set of components that have releases are:
+## Releases and Branching Strategy
 
-- `op-batcher`
-- `op-contracts`
-- `op-challenger`
-- `op-node`
-- `op-proposer`
+We maintain an `eigenda-develop` branch which is our main development branch which keeps a linear history containing new feature work, fixes, as well as upstream merges. To make downstream integrations easier, we create release-specific branches which contain cleaned up history of commits on top of a specific upstream release. For example, the second eigenda-fork release in the below picture would be named `op-batcher/v1.11.2-eigenda.2`, and will consist of a cleaned-up history of commits (one per feature/service pair) on top of the upstream [op-batcher/v1.11.2](https://github.com/ethereum-optimism/optimism/releases/tag/op-batcher%2Fv1.11.2) release. We will strive to make our releases on top of op [production releases](https://github.com/ethereum-optimism/optimism?tab=readme-ov-file#production-releases), unless an urgent fix is needed.
 
-All other components and packages should be considered development components only and do not have releases.
-
-### Development branch
-
-The primary development branch is [`develop`](https://github.com/ethereum-optimism/optimism/tree/develop/).
-`develop` contains the most up-to-date software that remains backwards compatible with the latest experimental [network deployments](https://docs.optimism.io/chain/networks).
-If you're making a backwards compatible change, please direct your pull request towards `develop`.
-
-**Changes to contracts within `packages/contracts-bedrock/src` are usually NOT considered backwards compatible.**
-Some exceptions to this rule exist for cases in which we absolutely must deploy some new contract after a tag has already been fully deployed.
-If you're changing or adding a contract and you're unsure about which branch to make a PR into, default to using a feature branch.
-Feature branches are typically used when there are conflicts between 2 projects touching the same code, to avoid conflicts from merging both into `develop`.
-
-## License
-
-All other files within this repository are licensed under the [MIT License](https://github.com/ethereum-optimism/optimism/blob/master/LICENSE) unless stated otherwise.
+![](./assets/fork-branching-and-releases.png)
```

Comment:
In this picture, `v1.11.2` contains `feat-c`, which isn't contained in the `eigenda-develop` history (there is only a `fix-c`)

Reply:
good catch! [490b63d](https://github.com/Layr-Labs/optimism/pull/37/commits/490b63d21b1dc244f11e48c8e1126b0dcf90d940)

Reply:
Sorry was still wrong. See [16bec65](https://github.com/Layr-Labs/optimism/pull/37/commits/16bec653039b2d6f1b3933584bfb95f09492fa51)